docs/src/reference/asciidoc/appendix/release-notes/8.0.0-rc1.adoc

```diff
@@ -14,6 +14,28 @@ https://github.com/elastic/elasticsearch-hadoop/pull/1808[#1808]
 - Update Gradle wrapper to 7.3
 https://github.com/elastic/elasticsearch-hadoop/pull/1809[#1809]
 
+- Update log4j version
+https://github.com/elastic/elasticsearch-hadoop/pull/1828[#1828]
+
+Serialization::
+- Add support for the `date_nanos` field type
+https://github.com/elastic/elasticsearch-hadoop/pull/1803[#1803]
+
 Spark::
 - Add support for Spark 3.1 and 3.2 
 https://github.com/elastic/elasticsearch-hadoop/pull/1807[#1807]
+
+[[bug-8.0.0-rc1]]
+[float]
+=== Bug fixes
+
+Core::
+- Resolve `saveToEs` saves case classes fields with `NULL` values
```

Comment:
These two in the `Core` section are actually both spark (I mislabeled them).

Reply:
Thanks. I moved them under Spark with https://github.com/elastic/elasticsearch-hadoop/pull/1863/commits/06ac86e13d052351608fef1b1c658cd14b1684aa.